src/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder('async_aws');
+        /** @var ArrayNodeDefinition $rootNode */
+        $rootNode = $treeBuilder->getRootNode();
+        $rootNode
+            ->fixXmlConfig('service')
+            ->children()
+                ->booleanNode('register_service')->info('If set to false, no service will be created.')->defaultTrue()->end()
```

Comment:
What's the purpose fot this? 

If people don't wan't services for a client, they just have to not Inject it? Autowire will remove not-used services at the end

Reply:
If someone wants to register a service without help of the bundle. They can disable the bundle's service to avoid conflicts with autowiring.